src/cuga/backend/cuga_graph/nodes/cuga_lite/helpers/bind_tools.py

```
@@ -0,0 +1,340 @@
+    resolve_bind_tools_fields,
+    resolved_runtime_model_name,
+)
+
+
+def _bind_tools_mode_from_settings() -> str:
+    try:
+        m = getattr(settings.advanced_features, "cuga_lite_bind_tools_mode", None)
+        if m is not None and str(m).strip():
+            return str(m).strip().lower()
+    except Exception:
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)